crates/turborepo-lib/src/commands/ls.rs

```diff
@@ -30,10 +31,42 @@ struct RepositoryDetails<'a> {
     packages: Vec<(&'a PackageName, &'a AnchoredSystemPath)>,
 }
 
+impl<'a> Serialize for RepositoryDetails<'a> {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        let mut state = serializer.serialize_struct("RepositoryDetails", 2)?;
+        state.serialize_field("package_manager", &self.package_manager)?;
```

Comment:
thought: I could go either way, but just wanted to mention that it came to mind that it might make sense, then, to also add this as an output line (or maybe on the same line as, "37 packages") to the pretty output.  Not a big deal, but it'd be nice if that pretty output had this information too.

Reply:
Added!